src/ado2gh/Commands/AddTeamToRepoCommand.cs

```diff
@@ -36,6 +36,10 @@ public AddTeamToRepoCommand(OctoLogger log, GithubApiFactory githubApiFactory) :
                 IsRequired = true,
                 Description = "The only valid values are: pull, push, admin, maintain, triage. For more details see https://docs.github.com/en/rest/reference/teams#add-or-update-team-repository-permissions, custom repository roles are not currently supported."
             };
+            var githubPat = new Option<string>("--github-pat")
+            {
+                IsRequired = false
```

Comment:
we should update the description on the command and/or on this option to explain that the user can provide the PAT either as an arg or env var. We should do this for all the commands.

Reply:
addressed in 2209c660837f608ba28a10ce49fbc00dad93887f